backends/c-sycl_intel/include/backend.hpp

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 //; eval_flags(c-sycl_intel): [dpcpp] -ldl -lpthread
-//; eval_flags(c-sycl_cuda): [/usr/local/dpcpp-cuda/bin/clang++] -ldl -I/usr/local/dpcpp-cuda/include/sycl -L/usr/local/dpcpp-cuda/lib -lsycl -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice -fsycl-unnamed-lambda -lpthread
+//; eval_flags(c-sycl_cuda): [/usr/local/dpcpp-cuda/bin/clang++] -O3 -ldl -I/usr/local/dpcpp-cuda/include/sycl -L/usr/local/dpcpp-cuda/lib -lsycl -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice -fsycl-unnamed-lambda -lpthread -Xsycl-target-backend=nvptx64-nvidia-cuda --cuda-gpu-arch=sm_75
 
```

Comment:
It doesn't require `--cuda-gpu-arch=sm_75` here since this compiling flag will only be use for codes that is not related to SYCL device codes. So you can safely remove it.

Reply:
Oh okay I see, thanks, I'll remove that